sw/airborne/modules/loggers/file_logger.c

```diff
@@ -106,56 +140,10 @@ void file_logger_stop(void)
 }
 
 /** Log the values to a csv file    */
-/** Change the Variable that you are interested in here */
 void file_logger_periodic(void)
 {
   if (file_logger == NULL) {
     return;
   }
-  static uint32_t counter;
-  struct Int32Quat *quat = stateGetNedToBodyQuat_i();
-
-#ifdef COMMAND_THRUST //For example rotorcraft
-  fprintf(file_logger, "%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d\n",
-          counter,
-          imu.gyro_unscaled.p,
-          imu.gyro_unscaled.q,
-          imu.gyro_unscaled.r,
-          imu.accel_unscaled.x,
```

Comment:
accel and mag are not part of this log anymore ?

Reply:
No, these were not relevant for the course. Instead I added the drone's velocity, attitude and rates. I assumed that people would re-add these measurements if they need them, as they will probably modify this file to their own needs anyway. It's not a problem to re-add them though, do you think I should do that?

Reply:
at some point, this file logger should be a generic API to write anything and the data to be written defined elsewhere, but that would be for another time